pilot.go

```diff
@@ -5,7 +5,6 @@ import (
 	"fmt"
 	"net"
 
-	"github.com/btcsuite/btcd/btcec/v2"
```

Comment:
Mistakenly removed?

Reply:
Sorry, it's Mistakenly removed, Fixed now, please recheck